.vale/styles/RedHat/CaseSensitiveTerms.yml

```diff
@@ -243,8 +243,6 @@ swap:
   RAW: raw
   Red Hat JBoss Data Grid|JDG: Red Hat Data Grid
   Red Hat JBoss EAP: Red Hat JBoss Enterprise Application Platform
-  Red Hat Network Satellite server: Red Hat Network Satellite Server
```

Comment:
Should this by updated to match https://redhat-documentation.github.io/supplementary-style-guide/#satellite-server?

Reply:
`Satellite server: Satellite Server`

Reply:
It's already there, on line 265